src/index.html

```diff
@@ -0,0 +1,133 @@
+                alt="">
+        </div>
+        <div id="search-area">
+            <form>
+                <input type="text" placeholder="Search">
+                <button id="search"><i class="fa fa-search"></i></button>
+            </form>
+            <button id="mic"><i class="fa fa-microphone" title="Search with your voice"></i></button>
+        </div>
+
+        <div id="right">
+            <i class="fa fa-video-camera" title="Create"></i>
+            <i class="fa fa-th" title="YouTube apps"></i>
+            <i class="fa fa-bell" title="Notifications"></i>
+            <span>M</span>
+        </div>
+    </nav>
+
+    <section>
+        <div class="left-side">
+            <div class="video-container">
+                <iframe width="855" height="438" src="https://www.youtube.com/embed/BqpqXYj589k" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            </div>
+            <span id="you-tags">#SlowDancing #Azawi #SwangzAvenue</span>
+            <p id="music-title">Azawi - Slow Dancing (Official Music Video)</p>
+
+            <div class="view-like-container">
+                <div class="views">41,630 views . Premiered Nov 26, 2021 </div>
+                <div class="likes-container">
+                    <span><i class="fa fa-thumbs-up"></i> 200k </span> <span><i class="fa fa-thumbs-down"></i>DISLIKE </span> <span><i class="fa fa-share"></i>SHARE </span> <span><i class="fa fa-scissors"></i>CLIP </span><span class="play-list"><img src="./playlist-plus-1-512.png"> SAVE</span> <span></span> <span><i class="fa fa-ellipsis-h"></i></span>
+                </div>
+            </div>
+            <hr>
+        </div>
+        <div class="right-side">
+            <div class="all-row">
+                <span class="all" title="All">All</span> <span title="Related">Related</span> <span title="Recently uploaded">Recently uploaded</span> <span  title="Watched">Watched</span>
+            </div>
+
+            <!-- FIRST VIDEO AREA -->
+            <div class="video-area">
+                <div class="sub-video-container">
+                    <img src="./mqdefault_6s.webp" alt="">
+                    <p>03:45</p>
+                </div>
+                <div class="text-container">
+                    <p class="text-title">
+                        Azawi - Slow Dancing <br>(Official Music Video)
+                    </p>
+                    <p class="artist-name">Azawi <span><img src="./noun_125647_cc-512.webp" alt=""></span></p>
+                    <p class="views-area">2M views . 6months ago</p>
+                </div>
+            </div>
+            <!-- SECOND VIDEO AREA -->
+            <div class="video-area">
+                <div class="sub-video-container">
+                    <img src="./secnd.webp" alt="">
```

Comment:
- [ ] You should consider giving a descriptive [`alt`](https://www.w3schools.com/tags/att_img_alt.asp) message.
A  descriptive **alt** attributes to image tags improves on the accessibility to blind individuals and also slow network. Kindly consider adding the `alt` attribute to `<img>` tag.